tests/contrib/aiopg/test.py

```diff
@@ -190,11 +191,37 @@ def test_patch_unpatch(self):
         assert spans, spans
         assert len(spans) == 1
 
-    @run_in_subprocess(env_overrides=dict(DD_SERVICE="mysvc"))
-    def test_user_specified_service(self):
+    @mark_asyncio
+    @run_in_subprocess(env_overrides=dict(DD_SERVICE="mysvc", DD_TRACE_SPAN_ATTRIBUTE_SCHEMA="v0"))
+    def test_user_specified_service_v0(self):
```

Comment:
There are two bugs here:

1. The existing behavior of the code base for `aiopg` is to set service name to `$DD_SERVICE` in the absence of a user-specified name.  This is a bug, because the expected behavior is to use the module name `aiopg` as the default.
1. The original unchanged test `test_user_specified_service` auto-succeeded no matter what asserts were in the function, as long as the asyncio yield code is included.  For example, in the unchanged code, your first line in the test could be `assert False` and it would still pass.

What I've done here is fix the testing bug (2) to document the current behavior and worked with you to open an issue for (1).  If/when (1) is addressed, the v0 test should be updated to once again be `!= "mysvc"`

Reply:
Ah, thanks for the re-run on it! Seems good!